src/app/routes/gateway.js

```diff
@@ -121,6 +121,12 @@ module.exports = (Router, Service) => {
       });
   });
 
+  Router.post('/gateway/user/findByEmail', basicAuth, async (req, res) => {
```

Comment:
Two things: 
- Let's aim to respect REST: 
   - Use a GET, not a POST, you are looking for a resource, not creating one
   - This route could be better `/gateway/users` with a query param called `email`
- Let's not ignore errors, use a try-catch and in case the find fails, return a 500 + 'Internal Server Error' message and on the other hand, log the error so we are aware of what's going on.